test/app/dftb+/bin/autotest2

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env sh
```

Comment:
Specifically for the sh scripts there might be an argument for either 1) keeping as /bin/sh or 2) switching to /usr/bin/env bash 
The original /bin/sh is explicitly getting POSIX compliance instead of whatever env serves up. The bash cases definitely should be via env.

Reply:
Does POSIX define the location of `sh`? In case a user provided POSIX compliant shell is used we want to pick this up.

Reply:
Done, using `bash` and `python3` now everywhere.